taxi/urls.py

```diff
@@ -46,6 +50,9 @@
     path(
         "drivers/<int:pk>/", DriverDetailView.as_view(), name="driver-detail"
     ),
+    path("drivers/create/", DriverCreateView.as_view(), name="driver-create"),
+    path("drivers/<int:pk>/update/", DriverUpdateView.as_view(), name="driver-update"),
+    path("drivers/<int:pk>/delete", DriverDeleteView.as_view(), name="driver-delete"),
```

Comment:
Don't forget to add `/` at the end of the URLs. It should be `"drivers/<int:pk>/delete/"` instead of `"drivers/<int:pk>/delete"`.